apps/old/widget/components/modals/propose/ProposalVisibilityInfoModal.jsx

```diff
@@ -0,0 +1,138 @@
+const { Button } = VM.require("${config_account}/widget/components") || {
+  Button: () => <></>,
+};
+const { Modal } = VM.require("${config_account}/widget/components.Modal") || {
+  Modal: () => <></>,
+};
+
+const ModalContainer = styled.div`
+  width: 480px;
+  height: 314px;
+  gap: 0;
+  border-radius: 20px;
+  background-color: #000000;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+`;
+const ModalHeading = styled.div`
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  text-align: center;
+  letter-spacing: 0.01em;
+  margin-top: 10px;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledButton = styled.div`
+  padding: 8px 16px;
+  border-radius: 10px;
+  margin-left: 8px;
+  cursor: pointer;
+  font-weight: 500;
+  font-family: inherit;
+  color: ${(props) => (props.variant === "copy" ? "#000000" : "#FFFFFF97")};
+  background-color: ${(props) =>
+    props.variant === "copy" ? "#FFAF51" : "#505050"};
+  transition: background-color 0.3s;
+  &:hover {
+    background-color: ${(props) =>
+      props.variant === "copy" ? "#FFC57F" : "#626262"};
+  }
+`;
+
+const TimerContainer = styled.div`
+  margin-top: -60px;
+  margin-left: 200px; /* Adjust margin as needed */
+  width: 48px;
+  height: 48px;
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100px;
+`;
+
+const Timer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 44px;
+  height: 44px;
+`;
+const Content = styled.div`
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  font-weight: 200;
+  text-align: center;
+  margin-top: 50px;
+  max-width: 450px;
+  margin: 0;
+  padding: 0 20px;
+`;
+const url =
+  "${alias_gateway_url}/${config_account}/widget/app?page=feed&tab=proposals";
+const ProposalVisibilityInfoModal = ({
+  open,
+  setCopied,
+  setInfoModalActive,
+  copied,
+  sdkCall,
+}) => {
+  const handleCopy = () => {
+    clipboard
+      .writeText(url)
+      .then(() => {
+        setCopied(true);
+        sdkCall(); // Call the function to create proposal
+        setTimeout(() => {
+          setCopied(false);
```

Comment:
Why do we need this timeout? Can’t we call it directly ?

Reply:
Removed the timeout.